constructr-coordination-consul/src/main/scala/com.tecsisa.constructr.coordination.consul/ConsulCoordinationSettings.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 TECNOLOGIA, SISTEMAS Y APLICACIONES S.L.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.tecsisa.constructr.coordination.consul
+
+import akka.actor.ActorSystem
+
+import scala.util.Try
+
+/**
+  * Created by domingueza on 24/03/2017.
+  */
```

Comment:
Please, do not use `@author` tags or the like. We embrace [Collective Code Ownership](http://www.extremeprogramming.org/rules/collective.html). We'll update our contributing guide including this.